manifest.json

```diff
@@ -14,9 +14,6 @@
     "64": "icons/icon64.png",
     "128": "icons/icon128.png"
   },
-  "externally_connectable": {
```

Comment:
I'm not sure if we really want to open the addon for communication with/from other addons. Sounds like a big security risk. This option is supported by all relevant browser, except firefox.
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/externally_connectable

I think it would be better to deny any communication with other addons or web pages.
I would just leave it as it is, or use it maybe in this way: `"externally_connectable": {},`

Reply:
Okay, confirmed. I've read again and again the whole documentation about `externally_connectable` and we should leave it as it is.